py/server/deephaven/_wrapper.py

```diff
@@ -21,7 +21,7 @@
 _di_wrapper_classes: Set[JObjectWrapper] = set()
 _has_all_wrappers_imported = False
 
-JPyObjectRefCountedNode = jpy.get_type('io.deephaven.server.plugin.python.PyObjectRefCountedNode')
+JLivePyObjectWrapper = jpy.get_type('io.deephaven.server.plugin.python.LivePyObjectWrapper')
```

Comment:
It does not appear like this type is exposed to the user.  Can it be made private? (`_` prefix)

Reply:
The whole file is already private, `_wrapper.py`, is that still advised?

Changed to both methods to use `Any` as suggested after discussing with Jianfeng.